web/frontend/src/lib/components/collapse.svelte

```diff
@@ -1,14 +1,20 @@
-<!--
-  Rendering embedded maps inside of a shadow dom is currently broken for MapLibreGL.
-  This Collapse component is taken from PRIME but removes all web component behavior.
-  Once we've completed the svelte migration it can be removed and replaced with the PRIME component.
--->
+<script lang='ts' context='module'>
+  export type StopCallback = (callback: () => void) => void
+</script>
+
 <script lang="ts">
 
 import { onMount, createEventDispatcher, tick } from 'svelte';
+import { Button } from '@viamrobotics/prime-core';
 
 export let title = '';
+export let crumbs = '';
 export let open = Boolean(localStorage.getItem(`rc.collapse.${title}.open`));
+export let hasStop = false
```

Comment:
Does this need to be an explicit prop? Or can we use the presence of a `stopCallback` to detect this?

Reply:
The problem is that the parent will always provide the callback, so there's no way for it to know if it should render this button. That is based on if the child card has a stop behavior, but the child shouldn't be directing the parent.